src/config.c

```diff
@@ -3329,6 +3344,7 @@ standardConfig static_configs[] = {
 
     /* Unsigned Long Long configs */
     createULongLongConfig("maxmemory", NULL, MODIFIABLE_CONFIG, 0, ULLONG_MAX, server.maxmemory, 0, MEMORY_CONFIG, NULL, updateMaxmemory),
+    createULongLongConfig("maxmemory-reserved", NULL, MODIFIABLE_CONFIG, 0, ULLONG_MAX, server.maxmemory_reserved, 0, MEMORY_CONFIG, NULL, updateMaxmemoryReserved),
```

Comment:
it would be appreciate if we support both percentage and absolute values, just like `maxmemory-clients`.

Reply:
I think we said we were going to punt the percent to a later point. If we make the config a LLONG_MAX, then we can support negative values in the future. So Maybe let's just do that?